src/logic/map_objects/tribes/production_program.cc

```diff
@@ -1393,7 +1454,8 @@ void ProductionProgram::ActMine::execute(Game& game, ProductionSite& ps) const {
 		//  there is a sufficiently high chance, that the mine
 		//  will still produce enough.
 		//  e.g. mines have chance=5, wells have 65
-		if (chance_ <= 20) {
+		if (depleted_chance_ <= game.world().get_resource(resource_)->max_amount() *
```

Comment:
Don't undestnad this change we need to have a small value her to show empty mines and notify the player while not notifiying the player about a depleted (normal) well

Reply:
Needs to be scaled up, just the same way as I commented above.

At the moment, `game.world().get_resource(resource_)->max_amount() == 20` for all 4 resources via their Lua configuration. So, no semantic change there either until we start implementing https://github.com/widelands/widelands/issues/3678 where the old code would cause a bug.

Reply:
this is not correct as this has nothing to do with the initial amount or the max amount. it is just an arbitrary value defined as threshold to declare an empty state on a building which mines something. the result is every building with a yield_depleted higher then 20 will never be empty --> no message in the news window, no empty animation. So mixing in the ressources amount would do no good here

Reply:
Thanks for explaining - I misunderstood the intent. Will change it back to 20.